### PR TITLE
Add tests for mc-fus library

### PR DESCRIPTION
**Pull request by annakocheshkova**
[AB#81102](https://msmobilecenter.visualstudio.com/Mobile-Center/_workitems/edit/81102)

### CODE REVIEW

File: test/commands/distribute/lib/mc-fus-uploader-test.ts [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+  constructor(name: string, filesize: number) {
+    this.name = name;
+    this.fileSize = filesize;
+  }
+
+  slice(start: number, end: number): Buffer {
+    throw new Error("Method not implemented.");
+  }
+
+  get fileName(): string {
```
Comment (MatkovIvan) [inline test/commands/distribute/lib/mc-fus-uploader-test.ts:30]: Remove it

File: test/commands/distribute/lib/mc-fus-uploader-test.ts [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+    return this.name;
+  }
+
+  get size(): number {
+    return this.fileSize;
+  }
+}
+
+const globalAsAny = global as any;
+globalAsAny.window = {};
```
Comment (MatkovIvan) [inline test/commands/distribute/lib/mc-fus-uploader-test.ts:40]: do we still need window?

Comment (annakocheshkova) [reply test/commands/distribute/lib/mc-fus-uploader-test.ts:36]: Yes it's used

File: test/commands/distribute/lib/mc-fus-uploader-test.ts [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+  McFusMessageLevel,
+  LogProperties,
+  IProgress,
+  McFusUploadState,
+  McFusFile,
+} from "../../../../src/commands/distribute/lib/mc-fus-uploader/mc-fus-uploader-types";
+import * as TypeMoq from "typemoq";
+import * as assert from "assert";
+import * as uuid from "uuid";
+import "abort-controller/polyfill";
```
Comment (MatkovIvan) [inline test/commands/distribute/lib/mc-fus-uploader-test.ts:13]: Remove polyfill

File: test/commands/distribute/lib/mc-fus-uploader-test.ts [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+
+import nock = require("nock");
```
Comment (MatkovIvan) [inline test/commands/distribute/lib/mc-fus-uploader-test.ts]: from?

File: test/commands/distribute/lib/mc-fus-uploader-test.ts [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+  constructor(name: string, filesize: number) {
+    this.name = name;
+    this.fileSize = filesize;
+  }
+
+  slice(start: number, end: number): Buffer {
+    throw new Error("Method not implemented.");
+  }
+
+  get size(): number {
```
Comment (MatkovIvan) [inline test/commands/distribute/lib/mc-fus-uploader-test.ts:30]: Can be removed if rename field `fileSize` to `size`

File: test/commands/distribute/lib/mc-fus-uploader-test.ts (hunk 2 above)
Comment (MatkovIvan) [reply test/commands/distribute/lib/mc-fus-uploader-test.ts:36]: It should not!

File: test/commands/distribute/lib/mc-fus-uploader-test.ts [hunk 6]
```diff
@@ -10,34 +10,26 @@ import {
...
-  get size(): number {
-    return this.fileSize;
-  }
 }
 
-const globalAsAny = global as any;
-globalAsAny.window = {};
 (URL as any).createObjectURL = () => {};
 
 // For the following two dependencies, we might want to move it to tests if we want to cover isBrowserSupported
```
Comment (MatkovIvan) [inline test/commands/distribute/lib/mc-fus-uploader-test.ts:32]: following two dependencies?

**Pull request merged**